Source/Initialization/WarpXInitData.cpp

```diff
@@ -330,10 +326,6 @@ WarpX::InitLevelData (int lev, Real /*time*/)
            if (lev > 0) {
               Bfield_aux[lev][i]->setVal(B_external_grid[i]);
               Bfield_cp[lev][i]->setVal(B_external_grid[i]);
-              if (fft_do_time_averaging) {
-                  Bfield_avg_aux[lev][i]->setVal(B_external_grid[i]);
-                  Bfield_avg_cp[lev][i]->setVal(B_external_grid[i]);
-              }
```

Comment:
Should we keep the initialization of the `avg_cp` data (both here for `Bfield_avg_cp` and below for `Efield_avg_cp`), since we keep the allocation of the `avg_cp` MultiFabs with mesh refinement? I mean that, even if the mesh refinement option is not yet fully tested, this initialization is probably correct and needed anyways. What do you think?

Reply:
Ah, yes, good point! Sorry: I went too fast here.